adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/CoapAdapterProperties.java

```diff
@@ -23,16 +23,52 @@
 public class CoapAdapterProperties extends ProtocolAdapterProperties {
 
     /**
-     * The default regular expression to split the identity into authority and tenant.
+     * The default regular expression to split the identity into authority and tenant. Value
+     * {@value #DEFAULT_ID_SPLIT_REGEX}.
      */
     public static final String DEFAULT_ID_SPLIT_REGEX = "@";
+    /**
+     * The default number of threads for the connector. Value {@value #DEFAULT_CONNECTOR_THREADS}.
+     */
+    public static final int DEFAULT_CONNECTOR_THREADS = 2;
+    /**
+     * The default number of threads for the coap protocol stack. Value {@value #DEFAULT_COAP_THREADS}.
+     */
+    public static final int DEFAULT_COAP_THREADS = 16;
+    /**
+     * The default number of threads for the dtls connector. Value {@value #DEFAULT_DTLS_THREADS}.
+     */
+    public static final int DEFAULT_DTLS_THREADS = 32;
```

Comment:
are these values supposed to be *per processor core*?

Reply:
No, per java-vm

Reply:
I don't think I can follow. Does this mean that the same value is supposed to be the right one for a VM with 1 (virtual) core and also for a VM with 64 cores?

Reply:
Yes, because of the current PSK limitation ;-(.

Reply:
And, iy you have 64 cores, just configure a larger value.

Reply:
So this value is indeed *per core*, right? If I have one core, 32 is a good value. If I have two cores, 64 is a good value? Or is there a certain minimum number of threads that we want to have?

Reply:
Currently I see this configuration value more as possibility to try out, what fits best. The actual default value is no based on too much experience.